providers/google/docs/operators/cloud/vertex_ai.rst

```diff
@@ -756,6 +756,10 @@ Interacting with Ray on Vertex AI Cluster
 To create a Ray cluster you can use
 :class:`~airflow.providers.google.cloud.operators.vertex_ai.ray.CreateRayClusterOperator`.
 
+Please note that you need to specify python_version and ray_version in :class:`~airflow.providers.google.cloud.operators.vertex_ai.ray.CreateRayClusterOperator`.
+Currently supported versions of ray package in ray cluster are: 2.9.3, 2.33, 2.42.
+For more information you can check: https://github.com/googleapis/python-aiplatform/blob/main/setup.py#L101
```

Comment:
I don't like that our docs reference to internal details of other packages - especially when we are referencing main branch where any commit can make this link not accurate. doesn't the library serve a user facing doc that we can reference instead?

Reply:
As I understand, this is specifically for users to see the limitations of the package they are trying to use, because without this it will be hard to determine what is wrong if the user will use the operator with unsupported versions of Ray.
For the link to the comment I agree, maybe better to refer to it in another way, more flexible one